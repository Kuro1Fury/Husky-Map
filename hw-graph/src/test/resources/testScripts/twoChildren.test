# Test that a node with two children work properly

CreateGraph A
AddNode A n1
AddNode A n2
AddEdge A n1 n1 e1
AddEdge A n1 n2 e2
ListNodes A
ListChildren A n1