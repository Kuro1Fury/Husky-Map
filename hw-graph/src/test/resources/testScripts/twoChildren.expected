# Test that a node with two children work properly

created graph A
added node n1 to A
added node n2 to A
added edge e1 from n1 to n1 in A
added edge e2 from n1 to n2 in A
A contains: n1 n2
the children of n1 in A are: n1(e1) n2(e2)